src/components/Tool/FixSendNewsletter/FixSendNewsletter.tsx

```diff
@@ -76,12 +76,20 @@ const FixSendNewsletter: React.FC<Props> = ({
   const { data, loading } = useGetInvalidNewsletterQuery({
     variables: { accountListId },
   });
+
+  let numberOfContacts = data?.contacts.nodes.length ?? 0;
+  if (data) {
+    const filtered = data.contacts.nodes.filter((node) => {
+      return !node.primaryPerson?.deceased;
+    });
```

Comment:
Could you filter out the deceased on the `GetInvalidNewsletterQuery` instead of doing it on this page multiple times?

Reply:
It seems like this should be possible, but is there any example of a nested filter? The `deceased` flag is on the `primaryPerson`, which is associated via a relationship. The `contactsFilter` does not have a way to filter these out on it directly.

Reply:
As I do research, for Apollo, at least, the `contactsFilter` would have to be modified to make this possible.

Reply:
This is something we should ask Andrew to add at a later date, this is okay for now.

Reply:
Created [MPDX-8053](https://jira.cru.org/browse/MPDX-8053) for this work.